apps/bbsync/query.py

```diff
@@ -389,3 +389,85 @@ def generate_comment(self):
                 "body": pending_comments.first().text,
                 "is_private": False,
             }
+
+    # TODO: test this
```

Comment:
When do you plan to test this? I would be for adding at least some simple test together with this while you still remember how is this functionality supposed to work :smile:

Reply:
This must have slipped through during the rewriting of the commits. Definitely shouldn't be there. It was there as a reminder because the associated tests belonged to a different commit due to the size of the changes. I'll review the old commits and make sure that the accompanying test is present in the PR.

Reply:
Fixed in commit [1fa690c](https://github.com/RedHatProductSecurity/osidb/pull/307/commits/1fa690cba505717ebc1353e8406395ea08d4a147)